backend/src/Request/Package/PackageCreateRequest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Request\Package;
+
+class PackageCreateRequest
+{
+    private $name;
+
+    private $cost;
+
+    private $note;
+
+    private $carCount;
+
+    private $city;
+
+    private $orderCount;
+
+    private $status;
+}
```

Comment:
Need an empty line after the ending of the class. Please, check all other classes.

Reply:
done